### PR TITLE
Updates to README

### PR DESCRIPTION
**Pull request by jpolton**
Task list for updates to README

- [x] Edit livljobs JAVA_HOME variable: `export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.372.b07-1.el7_9.x86_64/`
- [X] Add Mac JAVE_HOME variable 
- [x] Make note that the conda ENV set up currently does not work for ARM64 [no longer an issue as solved with #123]
- [X] Add wget instructions for benchmark data
- [X] Add explicit example on how to use benchmark data
- [X] Add explicit examples for making images with benchmark data
- [x] Fix layout of images

### CODE REVIEW

**Review by rdPatmore — COMMENTED** [review]
Would it better for an empty /benchmark dir to exist in the main?

**Comment by jpolton** [comment]
> Would it better for an empty /benchmark dir to exist in the main?

Good idea. Though I think GitHub doesn't like empty directories so in practise you have to add a README inside that says some thing like "This directory stores the benchmark data, obtainable from ..."

**Comment by jpolton** [comment]
It looks like `test_scripts/` is superseded by `plotting/`. @jdha  Is this true?

(The README references test_scripts in the plotting bit, but I think that this can be removed/replaced/tidied?)

**Comment by rdPatmore** [comment]
@jpolton, yes, I'd raised an issue on this earlier (#121). @jdha has it on the agenda for removal (#115). Mentions will also need to be removed in the README.md I guess.

**Comment by jdha** [comment]
I added a tick box to your list @jpolton about AMR64 ... hopefully once we migrate to cartopy this will help with installing on a Mac M1/2

File: README.md [hunk 1]
```diff
@@ -34,10 +34,16 @@ These are the steps to take to install pyBDY:
...
   ```
 
-- Make sure the Java Runtime Environment is set (e.g. livljobs\*):
+- Make sure the Java Runtime Environment is set E.g.:
 
   ```
-  export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.322.b06-1.el7_9.x86_64/
+  export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.372.b07-1.el7_9.x86_64/ # e.g. for livljobs\*
+  ```
+
```
Comment (jdha) [inline README.md:42]: You could also add the following to help the user (of *NIX architectures):
`readlink -f $(which java)`

File: README.md [hunk 2]
```diff
@@ -34,10 +34,16 @@ These are the steps to take to install pyBDY:
...
+  Or (downloading from https://jdk.java.net/20/)
+
+  ```
+  export JAVA_HOME=/Users/<username>/Downloads/jdk-20.0.1.jdk/Contents/Home/ # e.g. for mac OSX
   ```
```
Comment (jdha) [inline README.md:48]: Likewise for the Mac I think you can use:
`/usr/libexec/java_home`

File: README.md (hunk 1 above)
Comment (rdPatmore) [reply README.md:42]: what about replacing the static path with this command?

File: README.md (hunk 2 above)
Comment (rdPatmore) [reply README.md:48]: I'll update this.

File: README.md (hunk 1 above)
Comment (rdPatmore) [reply README.md:42]: Made some changes. See what you think.

File: README.md (hunk 2 above)
Comment (rdPatmore) [reply README.md:48]: Same on this one. Changes made.

File: screenshots/example_coords.png
Comment (jdha) [inline screenshots/example_coords.png:1]: Should we update with the new cartopy image that includes the baltic?

Comment (jpolton) [reply screenshots/example_coords.png:1]: done

**Review by jdha — APPROVED** [review]

**Review by rdPatmore — APPROVED** [review]
(y)

I have spotted a mention of bdy_coords_plot.py, which needs removing. I'll do that in a later merge.

**Pull request merged**